readthedocs/projects/tasks/builds.py

```diff
@@ -508,6 +508,10 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
                 build_pk=self.data.build['id'],
                 commit=self.data.build_commit,
                 status=status,
+                # Fix bug where we were generating relative links to the version
+                # with ``get_absolute_url``.
+                # https://github.com/readthedocs/readthedocs.org/issues/9791
```

Comment:
```suggestion
                # Fix bug where we were generating relative links to the version
                # with ``get_absolute_url``.
                # https://github.com/readthedocs/readthedocs.org/issues/9791
```

Reply:
@humitos there's no change here?

Reply:
Weird. I deleted the comment completely because it was not correct IMO